packages/manager/modules/enterprise-cloud-database/src/price/price.html

```diff
@@ -1,16 +1,16 @@
 <div data-ng-if="$ctrl.price" class="d-inline-flex">
     <div data-ng-if="$ctrl.asiaTouch.includes($ctrl.ovhSubsidiary)">
-        <span class="red bold" data-ng-bind="{{'price_price_gst_excl_label' | translate}}" data-translate-values="{ price: $ctrl.service.getPriceText($ctrl.price) }"></span>
-        (<span class="small" data-translate="price_price_gst_incl_label" data-translate-values="{ price: $ctrl.service.getPriceText($ctrl.price + $ctrl.tax) }"></span>)
+        <span class="red bold" data-ng-bind="{{'price_price_gst_excl_label' | translate}}" data-translate-values="{ price: $ctrl.getPriceText($ctrl.price) }"></span>
```

Comment:
Instead of `.red` maybe we should use one of UI-Kit color no ? https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/color . Should probably be checked with CX though

Reply:
The color has been removed